### ISSUE TITLE
Add documentation for the Redis cache driver

### ISSUE THREAD

**Issue by euantorano** [mybb/docs.mybb.com#192]
Since merging mybb/mybb#3958, we now support Redis as a cache driver. We need to add some documentation for it.

**Comment by Ben-MyBB** [comment]
I'm wondering if the best way would be to add a list of supported cache drivers and link to the relevant 3rd party documentation? E.G https://redis.io/documentation

**Comment by euantorano** [comment]
Yes, but also some documentation about how to set it up in `config.php` (which is easy really).

**Comment by Ben-MyBB** [comment]
I'll create a list with the relevant documentation and submit a PR over the next few days hoping to have it up in time for .23 release.
